recipe/run_test.py

```diff
@@ -0,0 +1,61 @@
+import subprocess
```

Comment:
this is new file, where did you get it from and what it actually test ? I do not see any pytest asserts here, just wondering ?

Reply:
I created it myself just to check if catboost is working and added there pip check too